pkg/apis/monitoring/v1/types.go

```diff
@@ -558,6 +558,12 @@ type ThanosSpec struct {
 	LogFormat string `json:"logFormat,omitempty"`
 	// MinTime for Thanos sidecar to be configured with. Option can be a constant time in RFC3339 format or time duration relative to current time, such as -1d or 2h45m. Valid duration units are ms, s, m, h, d, w, y.
 	MinTime string `json:"minTime,omitempty"`
+	// Volumes allows configuration of additional volumes on the output StatefulSet definition. Volumes specified will
+	// be appended to other volumes.
+	Volumes []v1.Volume `json:"volumes,omitempty"`
```

Comment:
It seems redundant with the `Volumes` field at the spec level. Maybe we need only `VolumeMounts` here?

Reply:
I will take a look on that

Reply:
Hi @simonpasquier, sorry for the late reply.

May you take a look again?

Reply:
Hi @simonpasquier, sorry for the inconvenience.

Any news about that?